packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { Story } from '@storybook/react';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import { LG } from '@zendeskgarden/react-typography';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import UserIcon from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+import ZendeskIcon from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+interface IStoryProps {
+  size: 'extraextrasmall' | 'extrasmall' | 'small' | 'medium' | 'large';
+  status: 'none' | 'away' | 'available';
+  badge: number;
+  surfaceColor: string;
+  backgroundColor: string;
+  foregroundColor: string;
+}
+
+export const Default: Story<IStoryProps> = ({
+  size,
+  status,
+  badge,
+  surfaceColor,
+  backgroundColor,
+  foregroundColor
+}) => {
+  const computedBadge = useMemo(() => {
+    if (badge === 0) {
+      return undefined;
+    }
+
+    if (badge > 9) {
+      return '9+';
+    }
+
+    return badge;
+  }, [badge]);
+
+  const computedStatus = useMemo(() => {
+    if (status === 'none') {
+      return undefined;
+    }
+
+    return status;
+  }, [status]);
+
+  return (
+    <Grid>
+      <Row>
+        <Col size="1">
+          <Row style={{ height: '10%' }}></Row>
+          <Row alignItems="center" style={{ height: '30%' }}>
+            <Col style={{ textAlign: 'center' }}>
```

Comment:
All the `textAlign` styles can be replaced with `Col` prop.

Reply:
Updated.